### PR TITLE
Adding new put for tasks

### PR DESCRIPTION
**Pull request by LuisTorres07**
Adding controller, service and validations to update tasks elements

### CODE REVIEW

File: src/main/java/com/stories/controller/StoriesController.java [hunk 1]
```diff
@@ -90,6 +90,16 @@ public void deleteTask(@Valid @PathVariable String id, @PathVariable String task
...
 		return storyService.updateStory(request, id);
 	}
+	
+	@ApiOperation(value = " PUT Task ", notes = "This operation will update a task from a story")
+	@ApiResponses({ @ApiResponse(code = 200, message = " Success operation "),
+					@ApiResponse(code = 404, message = " Task not found ") })
+	@ResponseStatus(value = HttpStatus.OK)
+	@PutMapping(value = "/{storyId}/tasks/{taskId}", produces = "application/json")
+	@ResponseBody
+	public TasksDomain updateTask(@Valid @RequestBody TasksDomain task, @PathVariable("storyId") String id, @PathVariable("taskId") String _id) throws Exception{
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/stories/controller/StoriesController.java]: maybe you could change the name to "updateTaskById"

Comment (VictorMunguiaEnciso) [inline src/main/java/com/stories/controller/StoriesController.java]: Please, verify any unnecessary blank space between the code lines and any unnecessary imports on this class please.

File: src/main/java/com/stories/service/StoriesServiceImpl.java [hunk 2]
```diff
@@ -126,6 +128,64 @@ public StoryDomain updateStory(StoryDomain storyDomain, String id) throws Except
 			throw new EntityNotFoundException(mensaggeDinamicValidation[0], mensaggeDinamicValidation[1]);
 		}
 	}
+
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/stories/service/StoriesServiceImpl.java:129]: Please, verify any unnecessary blank space between the code lines and any unnecessary imports on this class please.

File: src/main/java/com/stories/service/StoriesServiceImpl.java [hunk 3]
```diff
@@ -126,6 +128,64 @@ public StoryDomain updateStory(StoryDomain storyDomain, String id) throws Except
 			throw new EntityNotFoundException(mensaggeDinamicValidation[0], mensaggeDinamicValidation[1]);
 		}
 	}
+	
+	@Override
+	public TasksDomain updateTask(TasksDomain task, String id, String _id) throws Exception {
+		if(storiesRepository.existsById(id)) {//Validating that the story exists
+			task.set_id(_id);
+			TaskModel taskModel = storiesCustomRepository.getTaskById(id, _id).getUniqueMappedResult();
+			//Validating that the task exists
```
Comment (VictorMunguiaEnciso) [inline src/main/java/com/stories/service/StoriesServiceImpl.java]: Please remove the comments of this class.

**Review by VictorMunguiaEnciso — CHANGES_REQUESTED** [review]

**Comment by VictorMunguiaEnciso** [comment]
Please , fix the conflict

File: src/main/java/com/stories/controller/StoriesController.java (hunk 1 above)
Comment (LuisTorres07) [reply src/main/java/com/stories/controller/StoriesController.java]: Solved

Comment (LuisTorres07) [reply src/main/java/com/stories/controller/StoriesController.java]: Solved

File: src/main/java/com/stories/service/StoriesServiceImpl.java (hunk 2 above)
Comment (LuisTorres07) [reply src/main/java/com/stories/service/StoriesServiceImpl.java:129]: Solved

File: src/main/java/com/stories/service/StoriesServiceImpl.java (hunk 3 above)
Comment (LuisTorres07) [reply src/main/java/com/stories/service/StoriesServiceImpl.java]: Solved

**Pull request closed without merging**